src/ContinuousEnergyNeutron.tex

```diff
@@ -373,10 +386,14 @@ \subsubsection{\textsf{LDLW} \& \textsf{LDLWP} Blocks}\label{sec:LDLWBlock}\labe
 \end{tablenotes}
 \end{ThreePartTable}
 
-\textbf{All locators point to data \emph{relative} to \var{JED} (see \Sectionref{sec:DLWBlock}) in the \XSS\ array.} The \MT\ values are given in the \Block{MTR} for \Block{LDLW} or \Block{MTRP} for \Block{LDLWP}.
+The corresponding \MT\ values are given in the \Block{MTR} for \Block{LDLW} or \Block{MTRP} for \Block{LDLWP}.
+ 
+\textbf{All locators point to data \emph{relative} to \var{JED} (see \Sectionref{sec:DLWBlock}) in the \XSS\ array.}
+
+\subsubsection{\textsf{DLW} \& \textsf{DLWP} \& \textsf{DNED} Blocks}\label{sec:DLWBlock}\label{sec:DLWPBlock}\label{sec:DNEDBlock}
```

Comment:
I would prefer something like
```suggestion
\subsubsection{\textsf{DLW}, \textsf{DLWP}, and \textsf{DNED} Blocks}\label{sec:DLWBlock}\label{sec:DLWPBlock}\label{sec:DNEDBlock}
```
but maybe it's just a matter of personal taste

Reply:
I changed it, I don't like it either ;-)